anilist.py

```diff
@@ -183,19 +183,23 @@ def match_to_plex(anilist_series: List[AnilistSeries], plex_series_watched: List
         # Check if we have custom mappings for all seasons (One Piece for example)
         if len(plex_seasons) > 1:
             custom_mapping_season_count = 0
-            for plex_season in plex_seasons:
+            for index, plex_season in enumerate(plex_seasons):
                 season_mappings = retrieve_season_mappings(
                     plex_title, plex_season.season_number
                 )
                 matched_id = 0
                 if season_mappings:
                     matched_id = season_mappings[0].anime_id
                     if custom_mapping_seasons_anilist_id in (0, matched_id):
-                        plex_watched_episode_count_custom_mapping += plex_season.watched_episodes
                         custom_mapping_season_count += 1
-
                         custom_mapping_seasons_anilist_id = matched_id
 
+                        # Check if season should be added together
+                        if (plex_seasons[index - 1].last_episode + 1) - plex_season.first_episode <= 0:
```

Comment:
This can be written better.

```suggestion
                        if (index > 0 or plex_season.first_episode > plex_seasons[index - 1].last_episode):
```

The first condition is so that it doesn't check the first season against the last season. Alternatively, the first condition could be `index != 0` so it enters the `else`. For the first season, it doesn't matter whether `=` or `+=` is executed.

Reply:
Oops, should have been `index == 0` instead